chore(README): prepare for v0.9.0 (#507)
aquasecurity/trivy: README.md

[![License: Apache-2.0](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/aquasecurity/trivy/blob/master/LICENSE)
[![Docker image](https://images.microbadger.com/badges/version/aquasec/trivy.svg)](https://microbadger.com/images/aquasec/trivy "Get your own version badge on microbadger.com")

> A Simple and Comprehensive Vulnerability Scanner for Artifacts, Suitable for CI.

Author: I agree with you. I fixed it. https://github.com/aquasecurity/trivy/pull/507/commits/3839888f272294b5a7904eb1f6852cf2826a59f4

    + [Ignore the specified vulnerabilities](#ignore-the-specified-vulnerabilities)
    + [Specify cache directory](#specify-cache-directory)
    + [Clear caches](#clear-caches)
    + [Reset](#reset)
    + [Use lightweight DB](#use-lightweight-db)
  * [Client / Server](#client--server)
    + [Server](#server)
    + [Client](#client)
    + [Authentication](#authentication)
    + [Deprecated options](#deprecated-options)
- [Continuous Integration (CI)](#continuous-integration-ci)
  - [Travis CI](#travis-ci)
  - [CircleCI](#circleci)
  - [GitLab CI](#gitlab-ci)
  - [Authorization for Private Docker Registry](#authorization-for-private-docker-registry)
  * [Travis CI](#travis-ci)
  * [CircleCI](#circleci)
  * [GitLab CI](#gitlab-ci)
  * [Authorization for Private Docker Registry](#authorization-for-private-docker-registry)
- [Vulnerability Detection](#vulnerability-detection)
  - [OS Packages](#os-packages)
  - [Application Dependencies](#application-dependencies)
  * [OS Packages](#os-packages)
  * [Application Dependencies](#application-dependencies)
  * [Image Tar format](#image-tar-format)
  * [Data source](#data-source)
- [Usage](#usage)

> - [Comparison with other scanners](#comparison-with-other-scanners)

Reviewer: Did you intend to remove this section? (It would make sense to remove the Microscanner row)

Author: Yes, I don't follow the latest features of other scanners. So, I'm not sure this comparison is still true today. I'm thinking if we should remove the comparison section. What do you think?

Reviewer: I would leave it in, we can update it if someone complains that it's no longer true. 

We could also add in links to the comparison blog posts https://boxboat.com/2020/04/24/image-scanning-tech-compared/ and https://www.a10o.net/devsecops/docker-image-security-static-analysis-tool-comparison-anchore-engine-vs-clair-vs-trivy/

Author: I see. I reverted TOC and added the blogs.
https://github.com/aquasecurity/trivy/pull/507/commits/c26e6314795322f3dc9fbed6928b35e90de7b522

</details>

## Filesystem

> Scan your project including a lock file such as Gemfile.lock and package-lock.json.

Reviewer: ```suggestion
Scan a filesystem (such as a host machine, a virtual machine image, or an unpacked container image filesystem). Trivy will look for vulnerabilities based on lock files such as Gemfile.lock and package-lock.json.
```

```
$ trivy fs /path/to/project
```

Scan your container from inside the container.

```
$ docker run --rm -it alpine:3.11
/ # curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/master/contrib/install.sh | sh -s -- -b /usr/local/bin
/ # trivy fs /
```

## Embed in Dockerfile

> Scan your image in Dockerfile.

Reviewer: ```suggestion
Scan your image as part of the build process by embedding Trivy in the Dockerfile. This approach can be used to update Dockerfiles currently using Aqua’s [Microscanner][microscanner].
```

I’ll start deprecating Microscanner once this PR is released

  - Kaniko (https://github.com/GoogleContainerTools/kaniko)

> ## Data source

Reviewer: ```suggestion
## Data sources
```

  - https://github.com/RustSec/advisory-db

# Usage
Trivy has several sub commands, image, fs, repo, client and server.

```
NAME:
   trivy - A simple and comprehensive vulnerability scanner for containers

USAGE:
   trivy [global options] command [command options] image_name

VERSION:
   v0.9.0

COMMANDS:
   image, i          scan an image
   filesystem, fs    scan local filesystem
   repository, repo  scan remote repository
   client, c         client mode
   server, s         server mode
   help, h           Shows a list of commands or help for one command

GLOBAL OPTIONS:
   --quiet, -q        suppress progress bar and log output (default: false) [$TRIVY_QUIET]
   --debug, -d        debug mode (default: false) [$TRIVY_DEBUG]
   --cache-dir value  cache directory (default: "/Users/teppei/Library/Caches/trivy") [$TRIVY_CACHE_DIR]
   --help, -h         show help (default: false)
   --version, -v      print the version (default: false)

```

## Image

> `fs` and `repo` has the same options as `image`.

Reviewer: ```suggestion
`fs` and `repo` have the same options as `image`.
```
